核心/ksud.c

```diff
@@ -167,6 +167,8 @@ int ksu_handle_execveat_ksud(int *fd, struct filename **filename_ptr,
 				char first_arg[16];
 				#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0)
 				strncpy_from_user_nofault(first_arg, p, sizeof(first_arg));
+                                #elif (LINUX_VERSION_CODE >= KERNEL_VERSION(4, 14, 0) && LINUX_VERSION_CODE <= KERNEL_VERSION(5, 8, 0))
+				strncpy_from_user(first_arg, p, sizeof(first_arg));
 				#else
 				strncpy_from_unsafe_user(first_arg, p, sizeof(first_arg));
```

Comment:
I think change `strncpy_from_unsafe_user` to `strncpy_from_user` is ok.

Reply:
Should